bigquery/bigquery_create_dataset_iam/main.tf

```diff
@@ -15,6 +15,7 @@
  */
 
 
+# [START bigquery_bigquery_create_dataset_iam_parent_tag]
```

Comment:
@betsy-lichtenberg thinking out loud here. Technically speaking `bigquery.googleapis.com` part is already done before CICD test-setup.

If we just remove  this `google_project_service`, the existing region tags will serve as parent tags. WDYT ?

Reply:
I have added as reviewer for https://github.com/terraform-google-modules/terraform-docs-samples/pull/437